components/Toast.tsx

```diff
@@ -0,0 +1,56 @@
+import toast from 'react-hot-toast';
+import { useTranslation } from 'next-i18next';
+
+const Toast = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <div className='pointer-events-auto flex w-full max-w-md rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-10'>
+      <div className='w-0 flex-1 p-3'>
+        <div className='flex items-start'>
+          <div className='flex-shrink-0 pt-0.5'></div>
+          <div className='ml-3 flex-1'>{children}</div>
+        </div>
+      </div>
+      <div className='flex border-l border-gray-200'>
+        <button
+          onClick={() => toast.dismiss()}
```

Comment:
1. See a delay when clicking on close. We could consider using `toast.remove` instead which removes toast instantly.
2. I think it's better to pass along the toastId while closing the toast. Else in the scenario where we have multiple toasts stacked up, closing one will also close others.

Reply:
- Agreed. I already noticed there is a delay while closing the toast. It is a known issue with this package.
- Yes, that can be an issue if there are multiple toasts, not sure how to access the id here

Reply:
> not sure how to access the id here 

The custom function can access the current toast (See Render JSX custom content
 in https://react-hot-toast.com/docs/toast). Haven't tested it.
```jsx
toast.custom((t) => <ErrorToast message={message} toastId={t.id}/>
```

Reply:
I have restricted it to 2 toasts now, an error and success

Reply:
@niwsa the delay is the animation right?

Reply:
Yup but I think since we use the custom component, the animation is absent.